config/defaults.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
+  taskcluster: {
+    queueBaseUrl: 'https://queue.taskcluster.net/v1',
+    authBaseUrl: 'https://auth.taskcluster.net/v1',
+    credentials: {
+      clientId:     undefined,
+      accessToken:  undefined,
+    }
+  },
+  pulse: {
+    username: undefined,
+    password: undefined,
+  },
+  provisioner: {
+    id:                   'aws-provisioner2',
+    workerTypeTableName:  'AwsWorkerTypes',
+    publishMetaData:      'false',
+    statsComponent:       'aws-provisioner2',
+    awsKeyPrefix:         'aws-provisioner2-managed:',
+    iterationInterval:    1000 * 10,
+    awsInstancePubkey:    undefined,
+    allowedRegions:       'us-west-1', // comma seperated list
+  },
+  server: {
+    publicUrl:  'https://localhost',
+    port:       5556,
+    env:        'development',
+    forceSSL:   false,
+    trustProxy: false,
+  },
+  azure: {
+    accountName: undefined,
+    accountKey:  undefined,
+  },
+  aws: {
+    accessKeyId: undefined,
+    secretAccessKey: undefined
```

Comment:
You might want to consider specifying API version... I don't know how important it is. We just do everywhere else to lock things down (bumping the API version is fairly trivial after all).